### PR TITLE
Add reset button

### PR DESCRIPTION
**Pull request by RupengLiu**
Close #233 
image
image
image

### CODE REVIEW

File: src/services/provisioningService.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    { name: "Content-Type", value: "application/json" },
+                    { name: "Authorization", value: accessToken }
+                ],
+            };
+            const response = await this.sendRequest(request);
+            const contentTypes = Object.values(response["value"]);
+            for (const contentType of contentTypes) {
+                const contentTypeName = contentType["name"];
+                const curReq: HttpRequest = {
+                    url: `${managementApiUrl}/contentTypes/${contentTypeName}/contentItems?api-version=2018-06-01-preview`,
```
Comment (azaslonov) [inline src/services/provisioningService.ts]: `api-version=2018-06-01-preview` - let take this from settings.

File: src/components/content/content.ts [hunk 2]
```diff
@@ -23,10 +23,20 @@ export class PublishingWorkshop {
...
             this.viewManager.notifySuccess("Publishing", `The website is being published...`);
-            this.viewManager.closeWorkshop("publishing-workshop");
+            this.viewManager.closeWorkshop("content-workshop");
         }
         catch (error) {
             this.viewManager.notifyError("Publishing", `Unable to schedule publishing. Please try again later.`);
         }
     }
+
+    public async unpublish(): Promise<void> {
```
Comment (azaslonov) [inline src/components/content/content.ts]: `unpublish()` should it be `reset()`?

File: src/services/provisioningService.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            this.viewManager.showToolboxes();
+        }
+        catch (error) {
+            throw error;
+        }
+    }
+
+    private async cleanupContent(): Promise<void> {
+        const managementApiUrl = await this.getManagementUrl();
+        const url = `${managementApiUrl}/contentTypes?api-version=2018-06-01-preview`;
```
Comment (azaslonov) [inline src/services/provisioningService.ts]: `api-version=2018-06-01-preview` we should take if from config.

**Comment by mikebudzynski** [comment]
Can you rename the menu title "Publishing" to "Portal operations"?

**Comment by RupengLiu** [comment]
Sure

**Comment by RupengLiu** [comment]
> Can you rename the menu title "Publishing" to "Portal operations"?



> Can you rename the menu title "Publishing" to "Portal operations"?

sure

**Pull request merged**